src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,50 @@
+.page {
+  min-height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
```

Comment:
you declared a variable for gap, use it here

Reply:
Thank you)